tests/frameworks/test_pyramid.py

```diff
@@ -1,319 +1,337 @@
 # (c) Copyright IBM Corp. 2021
 # (c) Copyright Instana Inc. 2020
 
-import unittest
-
+import pytest
 import urllib3
+from typing import Generator
 
-import tests.apps.pyramid_app
-from ..helpers import testenv
+import tests.apps.pyramid.pyramid_app
+from tests.helpers import testenv
 from instana.singletons import tracer, agent
+from instana.span.span import get_current_span
 
 
-class TestPyramid(unittest.TestCase):
-    def setUp(self):
-        """ Clear all spans before a test run """
+class TestPyramid:
+    @pytest.fixture(autouse=True)
+    def _resource(self) -> Generator[None, None, None]:
```

Comment:
You are not using yield but the return type is Generator[None, None, None]. Is this correct?